cpp/ps0004/ps0004_solv.cpp

```diff
@@ -4,30 +4,22 @@ using namespace std;
 
 int main()
 {
-    int x , counter;
+    // prompting user for number 1/1.5
+    float num;
+    cout << "\nPlease enter a number between 32 and 126:\t";
+    cin >> num;

```

Comment:
- [x] declared a variable to store the user input
- [ ] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input